ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -2313,6 +2313,12 @@ export const WorkspaceEdit = fp.flow(
                   {loading && (
                     <SpinnerOverlay overrideStylesOverlay={styles.spinner} />
                   )}
+                  <div style={{ marginBottom: '1rem' }}>
+                    <b>
+                      Note: this workspace will take approximately one minute to
```

Comment:
Should we have upper case T for Note: This?

Reply:
I think it's more natural without.